packages/playwright-core/src/server/debugController.ts

```diff
@@ -144,9 +147,17 @@ export class DebugController extends SdkObject {
   }
 
   async highlight(params: { selector?: string, ariaTemplate?: string }) {
+    // Assert parameters validity.
+    if (params.selector)
+      unsafeLocatorOrSelectorAsSelector(this._sdkLanguage, params.selector, 'data-testid');
```

Comment:
Who catches the error here?

Reply:
debugControllerDispatcher will try catch and return the protocol error.